### DISCUSSION TITLE
Casks/ireadfast.rb. macOS 32bit

### DISCUSSION THREAD

**Discussion by andrewcrook** [Homebrew/discussions#2935] [Casks] [answered]
I just wanted to report that 
[iReadFast](https://github.com/Homebrew/homebrew-cask/blob/HEAD/Casks/ireadfast.rb)
Is no longer supported on many Macs because its 32bit.

Is there away to exclude casks that do not meet the basic requirments of the machine brew is running on?

**Comment by neutric** [comment, marked as answer]
The last macOS supporting 32-bit apps is Mojave, iirc. With `depends_on macos: "<= :mojave"` you can exclude this cask from being available to newer OS version not supporting it.

**Reply by andrewcrook** [reply]
thanks i will look into it

**Reply by neutric** [reply]
@andrewcrook I wasn't clear, sorry. `depends_on macos: "<= :mojave"` would have to be added to the cask definition via a PR. Let me know should you need help!

**Reply by andrewcrook** [reply]
@neutric oh yes I understood thanks I have done it. I had an issue at first with order but sorted it.

[Cask ireadfast: 32-bit macOS dependency add depends_on #118911](Cask ireadfast: 32-bit macOS dependency add depends_on #118911)
